### ISSUE TITLE
release

### ISSUE THREAD

**Issue by Harut** [lunant/mockcache#8]
Could you please release a latest version to pypi? It will be useful for me. Thanks!

**Comment by cclauss** [comment]
Yes please.  #6 was key to being able to install on Python 3 but that fix is not in [the PyPI release](https://pypi.org/project/mockcache).

@dahlia @youknowone @lunant @LouisPlisso  Can we please get a new release of the current code to PyPI.  If not, to you have a suggestion of an alternative?
